x-pack/legacy/plugins/epm/common/types.ts

```diff
@@ -108,28 +124,19 @@ export interface Dataset {
   type: string;
 }
 
-export interface RegistryPackage {
-  name: string;
-  version: string;
-  description: string;
-  readme?: string;
-  icon: string;
-  requirement: RequirementsByServiceName;
-  title?: string;
-  screenshots?: ScreenshotItem[];
-  assets: string[];
-  datasets: Dataset[];
+// some properties are optional in Registry responses but required in EPM
+// internal until we need them
+interface PackageAdditions {
+  title: string;
```

Comment:
Make `title` a required property since we ensure we have one in https://github.com/jfsiii/kibana/blob/575467c5eb6013b640f5f56589c3b809f7355bd7/x-pack/legacy/plugins/epm/server/packages/get.ts#L68